tests/functional/sign_up.js

```diff
@@ -120,6 +120,55 @@ define([
         .end();
     },
 
+    'sign up, verify and sign out of two accounts, all in the same tab, then sign in to the first account': function () {
+      // https://github.com/mozilla/fxa-content-server/issues/2209
+      var secondEmail = TestHelpers.createEmail();
+      var self = this;
+      return fillOutSignUp(this, email, PASSWORD)
+        .then(function () {
+          return testAtConfirmScreen(self, email);
+        })
+        .then(function () {
+          return FunctionalHelpers.getVerificationLink(email, 0);
+        })
+        .then(function (verificationLink) {
+          return self.remote.get(require.toUrl(verificationLink));
+        })
+        .findByCssSelector('#fxa-settings-header')
```

Comment:
Nit: can you add a space above `findBy*` and below `.end` calls so the grouping is a bit more obvious?